.github/workflows/build_and_test.yml

```diff
@@ -156,6 +179,12 @@ jobs:
             echo "Download the bundle at https://storage.googleapis.com/${{ inputs.GCP_BUCKET }}/${DATA_BASENAME}"
           fi
         fi
+
+        # Remove the container and the workspace to avoid any conflict with the next run.
+        echo github.workspace = ${{ github.workspace }}
+        #rm -rf ${{ github.workspace }}/*
+        #docker rm -f ${CONTAINER_NAME}
```

Comment:
So do you still need to clean up?

Reply:
see above

Reply:
Then can you add a small comment about it?